packages/noports_core/lib/src/sshnp/util/sshnp_initial_tunnel_handler/sshnp_openssh_initial_tunnel_handler.dart

```diff
@@ -1,14 +1,18 @@
 import 'dart:async';
 import 'dart:convert';
 
+import 'package:meta/meta.dart';
 import 'package:noports_core/src/common/io_types.dart';
 import 'package:noports_core/src/common/openssh_binary_path.dart';
 import 'package:noports_core/sshnp_foundation.dart';
 
 mixin SshnpOpensshInitialTunnelHandler on SshnpCore
     implements SshnpInitialTunnelHandler<Process?> {
   @override
-  Future<Process?> startInitialTunnel({required String identifier}) async {
+  Future<Process?> startInitialTunnel({
+    required String identifier,
+    @visibleForTesting ProcessStarter startProcess = Process.start,
+  }) async {
```

Comment:
Made startProcess an argument in this function, which can be overridden with a mock for testing